### DISCUSSION TITLE
Restrictions for bulding great temple

### DISCUSSION THREAD

**Discussion by pharon4ik** [Keriew/augustus#443] [Q&A] [answered]
Hello gentlemens, how i can remove restrictions for bulding great temple? Example if i want build than more 2 temple ?

**Comment by crudelios** [comment, marked as answer]
In `src/building/monument.h`

Change this line:

```
#define MAX_GRAND_TEMPLES_PER_CITY 2
```

To whatever you want the maximum to be.

Then build Augustus yourself.

Check [Building Julius](https://github.com/bvschaik/julius/wiki/Building-Julius) (it also works for Augustus) for details.

**Reply by crudelios** [reply]
There is now an option on the latest unstable where you can set the max number of grand temples per city.
